test/pkg/config/image_tags.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+type ImageTagsInterface interface {
+	GetImageTagPointerFromPersistenceType(imageType ImageType, persistenceType ImagePersistenceType) *string
+}
+
+type ImageType string
+
+const (
+	DataIndexImageType         ImageType = "data-index"
+	ExplainabilityImageType    ImageType = "explainability"
+	JobServiceImageType        ImageType = "jobs-service"
+	ManagementConsoleImageType ImageType = "mgmt-console"
+	TaskConsoleImageType       ImageType = "task-console"
+	TrustyImageType            ImageType = "trusty"
+	TrustyUIImageType          ImageType = "trusty-ui"
+)
+
+type ImagePersistenceType string
+
+const (
+	InfinispanPersistenceType ImagePersistenceType = "infinispan"
+	MongoDBPersistenceType    ImagePersistenceType = "mongodb"
+	NoPersistenceType         ImagePersistenceType = ""
```

Comment:
Shouldn't it be `ephemeral` in this case?

Reply:
true. updated